README.md

```diff
@@ -15,6 +15,11 @@ This documentation is for developers interested in using this .NET client to int
 * [Generate a preview template](#generate-a-preview-template)
```

Comment:
Not sure what we could add here, I think in VS there's a number of steps to add a another nuget source so could add a link describing it rather than inflating the docs.